db-migration/mysql/README.md

```diff
@@ -0,0 +1,179 @@
+# MySQL database migration using Python
+
+The MySQL database migration script allows you to migrate MySQL databases from one server to another. This is applicable to any platform:
+
+* IBM classic infrastructure to IBM Cloud VPC
+* On-premises to IBM Cloud VPC
+* Other cloud service providers to IBM Cloud VPC
+
+## Prerequisites
+
+Review the following prerequisites before you begin your migration:
+
+1. Set up an IBM Cloud Object Storage bucket.
+2. Make sure that you have access to write to IBM Cloud Object Storage bucket.
+3. Make sure that you have connection to the source and target server from your system.
+4. Ensure that Python3 (version 3.0) and Pip3 are installed on your system.
+
+## Cloning and running the database migration script
```

Comment:
Do we need to call out where this script can be executed from (ie, separate device, serverless function)?

Reply:
No, its already mentioned as users system.